src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type Prop = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Prop> = ({ user }) => {
+  return (
+    <p className="postList__user">
+      {`${user?.name} ${user?.email}`}
```

Comment:
don't understand why you set fields name and email of user as unnecessary. As for me if you have Post you should have author of it. Put anonymous values as default if you want hide something